jaxrs-tck/pom.xml

```diff
@@ -119,14 +119,6 @@
             </plugin>
             <plugin>
                 <artifactId>maven-source-plugin</artifactId>
-                <executions>
```

Comment:
This was required due to https://issues.apache.org/jira/browse/MSOURCES-121. The `jar-no-fork` goal is defined as an execution in the root POM.